src/cdk-lib/bedrock/guardrails/guardrails.ts

```diff
@@ -0,0 +1,552 @@
+    const defaultBlockedOutputsMessaging = 'Sorry, I am unable to answer your question because of our usage policy.';
+
+    // ------------------------------------------------------
+    // CFN Props - With Lazy support
+    // ------------------------------------------------------
+    let cfnProps: bedrock.CfnGuardrailProps = {
+      name: this.name,
+      description: props.description,
+      kmsKeyArn: props.kmsKey?.keyArn,
+      blockedInputMessaging: props.blockedInputMessaging ?? defaultBlockedInputMessaging,
+      blockedOutputsMessaging: props.blockedOutputsMessaging ?? defaultBlockedOutputsMessaging,
+      // Lazy props
+      contentPolicyConfig: this.generateCfnContentPolicyConfig(),
+      contextualGroundingPolicyConfig: this.generateCfnContextualPolicyConfig(),
+      topicPolicyConfig: this.generateCfnTopicPolicy(),
+      wordPolicyConfig: this.generateCfnWordPolicyConfig(),
+      sensitiveInformationPolicyConfig: this.generateCfnSensitiveInformationPolicyConfig(),
+    };
+
+    // Hash calculation useful for versioning of the guardrail
+    this.hash = md5hash(JSON.stringify(cfnProps));
+
+    // ------------------------------------------------------
+    // L1 Instantiation
+    // ------------------------------------------------------
+    this.__resource = new bedrock.CfnGuardrail(this, 'MyGuardrail', cfnProps);
+
+    this.guardrailId = this.__resource.attrGuardrailId;
+    this.guardrailArn = this.__resource.attrGuardrailArn;
+    this.guardrailVersion = this.__resource.attrVersion;
+  }
+
+  // ------------------------------------------------------
+  // METHODS
+  // ------------------------------------------------------
+  /**
+   * Adds a content filter to the guardrail.
+   * @param filter The content filter to add.
+   */
+  public addContentFilter(filter: filters.ContentFilter): void {
+    this.contentFilters.push(filter);
+  }
+
+  /**
+   * Adds a PII filter to the guardrail.
+   * @param filter The PII filter to add.
+   */
+  public addPIIFilter(filter: filters.PIIFilter): void {
+    this.piiFilters.push(filter);
+  }
+
+  /**
+   * Adds a regex filter to the guardrail.
+   * @param filter The regex filter to add.
+   */
+  public addRegexFilter(filter: filters.RegexFilter): void {
+    this.regexFilters.push(filter);
+  }
+
+  /**
+   * Adds a denied topic filter to the guardrail.
+   * @param filter The denied topic filter to add.
+   */
+  public addDeniedTopicFilter(filter: filters.DeniedTopic): void {
+    this.deniedTopics.push(filter);
+  }
+
+  /**
+   * Adds a contextual grounding filter to the guardrail.
+   * @param filter The contextual grounding filter to add.
+   */
+  public addContextualGroundingFilter(filter: filters.ContextualGroundingFilter): void {
+    this.contextualGroundingFilters.push(filter);
+  }
+
+  /**
+   * Adds a word filter to the guardrail.
+   * @param filter The word filter to add.
+   */
+  public addWordFilter(filter: string): void {
```

Comment:
There is also a way through the console to add a word filter from a file stored in S3 when using the console, we might consider adding it
![image](https://github.com/user-attachments/assets/58906e58-8aa4-4793-8933-3f1b7cd693d1)

Reply:
Unfortunately there is no [property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-bedrock-guardrail-wordpolicyconfig.html) exposed through CloudFormation that allows that 😔.

Reply:
ACK, resolving